kernel/src/fs/path.c

```diff
@@ -49,8 +51,7 @@ struct DirEntry *path_get_direntry(struct Vfs *vfs, Path *path, enum VfsErr *err
         pathpointer = path->array + 1;
 
     } else {
-        kprintf("Inode lookup only supports absolute path right now.");
-        panic();
+        FATAL("Inode lookup only supports absolute path right now.");
```

Comment:
the vfs isn't even close to done because we stopped developing it when we found out we had to fix 20000 more things before it would ever work. It indeed is todo but I don't think it's really worth it adding it now as we're going to look over that code anyway. The only reason this line changed here is because we replaced all printf+panic with a fatal.

Reply:
I've created an issue for creating/finishing the tmpfs/vfs: #47

Reply:
Alright, thanks!